internal/runners/activate/checkout.go

```diff
@@ -72,3 +81,21 @@ func (r *Checkout) Run(namespace string, targetPath string) error {
 
 	return nil
 }
+
+func getLanguage(owner, project string) (string, error) {
+	modelLanguage, fail := model.DefaultLanguageForProject(owner, project)
+	if fail != nil {
+		return "", fail
+	}
+
+	language := modelLanguage.Name
+	if strings.ToLower(modelLanguage.Name) == "python" {
+		version, err := semver.Parse(modelLanguage.Version)
+		if err != nil {
+			return language, err
+		}
+		language = modelLanguage.Name + strconv.FormatUint(version.Major, 10)
```

Comment:
I think you basically just need http://github.com/ActiveState/cli/blob/77e2701270db9b9789e00e1274af701a5fa1443c/internal/language/language.go#L115-L115

Reply:
The name value we get from the API for both versions of python is just `python`. The `MakeByName` function expects `python2` or `python3` for a successful lookup.

Reply:
I see. In that case let's keep this logic but move it next to `MakeByName` and name it `MakeByNameAndversion`